chapters/data-processing.tex

```diff
@@ -0,0 +1,725 @@
+with data on the same observations in other datasets, 
+such as other rounds of data collection and different data sources.
+Therefore, you must keep a canonical dataset of all observations in your data.
+This \textbf{master dataset}\sidenote{\url{
+		https://dimewiki.worldbank.org/Master\_Data\_Set}}
+is discussed in detail in Chapter 3, 
+and contains a record of all units ever encountered for each relevant level of observation,
+as well as their corresponding unique identifiers.
+
+
+% Merchand
+Checking for duplicated entries and making sure that each observation has a unique identifier are usually part of data quality monitoring.
+The lack of a unique identifier should be addressed as soon as data is received,
+so you can keep track of observations.
+\texttt{ieduplicates} and \texttt{iecompdup},
+two Stata commands included in the \texttt{iefieldkit}
+package\index{iefieldkit},\sidenote{\url{https://dimewiki.worldbank.org/iefieldkit}}
+create an automated workflow to identify, correct and document
+occurrences of duplicate entries.
+One advantage of using \texttt{ieduplicates}\sidenote{\url{
+		https://dimewiki.worldbank.org/wiki/ieduplicates}} 
+to correct duplicated entries is that it creates \textit{duplicates reports}
+that record which corrections made and why.
+Even if you are not using this command,
+it is important to include similar records in the \texttt{Documentation} folder.
+
+\subsection{Tidying raw data}
+
+Though raw data can be acquired in all shapes and sizes,
+development data is most commonly transferred as one or multiple tables.\sidenote{\textbf{Table:}
+	a two-dimensional dataset formed by rows and columns. 
+	Also called ``rectangular data''.
+	Examples of non-rectangular data are a text, or files, such as images.}
+Similarly, these tables can organize information in multiple ways,
+and not all of them result in easy-to-handle datasets.
+Fortunately, a vast literature of database management has identified the format
+that makes interacting with the data as easy as it can be.
+When applied to statistics, we call data in such format \textbf{tidy}.\sidenote{\textbf{Tidy data:}
+	Datasets where each column represents one variable,
+	each row represents one observation, 
+	and all variables in a table have the same unit of observation.}
+A dataset is tidy when each column represents one variable,\sidenote{\textbf{Variable:} 
+	the collection of all data points that measure the same attribute.}
+each row represents one observation, 
+and all variables in a table have the same unit of observation.
+Every other format is \textit{untidy} or just \textit{messy}.
+This may seem trivial, but raw data, 
+and raw survey data in particular,
+is rarely received in a tidy format.
+To contribute to the confusion, 
+Stata users typically call each column a ``variable'', 
+and each row an ``observation'',
+even though they are not necessarily the same thing.
```

Comment:
I have heard users from all languages call a row an observation interchangeably. So I am not sure we should say this.

I still think there is a point, as a column in Stata is not only called variable by its users, that is what it is officially called. So make the point about column/variable in Stata, but I do not think we can say that Stata users call a row and observation more or less than other people.

Reply:
done in 39a1b40